### DISCUSSION TITLE
GitHub wiki

### DISCUSSION THREAD

**Discussion by radarhere** [python-pillow/Pillow#6218] [General]
GitHub provides a "wiki" feature. python-pillow has made minimal use of this. There are [GSOC ideas from 2014](https://github.com/python-pillow/Pillow/wiki/GSOC-Ideas) (edit: archived [here for posterity](https://github.com/python-pillow/Pillow/files/8501071/GSOC_Ideas.md)), and that's about it.

Until 4 days ago that is, when @artemkaw [added a banner to the home page](https://github.com/python-pillow/Pillow/wiki/Home/2288f7dcc3ec5463456e64097d3ac8aa14193501). I was surprised to discover this. The user no longer has a GitHub profile, so I can't open a dialog with them about that.

I'm starting this discussion not to talk about the banner itself, but rather the fact that an unknown user silently made a change. To me, it seems like we are at risk of vandalism. Although perhaps this is exactly the point of the "wiki" feature, that it is a space for open documentation.

So I thought I would ask - are we comfortable with the fact that anyone can come along and make a change like that?

Alternatively, we could [restrict editing of the wiki to collaborators only](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis), or [disable it completely](https://docs.github.com/en/communities/documenting-your-project-with-wikis/disabling-wikis) since we house our documentation at https://pillow.readthedocs.io/ instead.

**Comment by hugovk** [comment]
I'd be fine turning the wiki off, it's clearly not being used, and one less tab at the top. Wikis can be really useful but I normally turn then off for GitHub projects. 

We should preserve the GSOC info somewhere, an issue would be fine, even this one. I doubt we need the edit history. (Interesting to see GitHub wikis are Git repos too!)

**Reply by radarhere** [reply]
I've uploaded a copy of the GSOC info to the original post here.
